apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -147,6 +149,23 @@ class SectionProgress extends Component {
       (currentView === ViewType.SUMMARY || currentView === ViewType.DETAIL);
     const standardsStyle =
       currentView === ViewType.STANDARDS ? styles.show : styles.hide;
+
+    // Reorder coursesWithProgress so that the current section is at the top and other sections are in order from newest to oldest
+    // Also adds "(Current Section)" label to the course version in the currently viewed section.
+    if (coursesWithProgress[0].id !== sectionVersionId) {
+      let currentCourse;
+      currentCourse = coursesWithProgress.splice(
+        coursesWithProgress.findIndex(course => {
+          return course.id === sectionVersionId;
+        }),
+        1
+      )[0];
+      currentCourse.display_name =
+        '(Current Section) ' + currentCourse.display_name;
+      coursesWithProgress.push(currentCourse);
+      coursesWithProgress.reverse();
+    }
```

Comment:
Seems cleaner this way (you'd then need to pass `reorderedCourses` to the `UnitSelector` component).

```suggestion
    const reorderedCourses = [
      ...coursesWithProgress.filter(course => course.id === sectionVersionId),
      ...coursesWithProgress.filter(course => course.id !== sectionVersionId),
    ];
```

Reply:
Yep. That's better. I'll replace it.